Blokaroka.cpp

```diff
@@ -8,6 +8,13 @@ Config CONFIG;
 int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance,
     PSTR pstrCmdLine, int iCmdShow)
 {
+    const char UniqueMutex[] = "one_Instance_only";
+    Handle hHandle = CreateMutex(NULL, TRUE, UniqueMutex);

```

Comment:
In order to build this needs to have Handle replaced with the all caps HANDLE, I'm not sure how to make that change and commit the pull request at the same time though!